packages/design-system-docs/src/pages/components/TextField/LabelMask.example.jsx

```diff
@@ -0,0 +1,44 @@
+import { TextField } from '@design-system';
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+const Example = () => {
+  return (
+    <div>
+      <TextField
+        label="Phone number"
+        labelMask="phone"
+        inputMode="tel"
+        name="phone_example"
+        type="tel"
+        helpText="(123) 456-7890"
```

Comment:
Just noticing this prop and it'll probably break a site build, but it reminded me of a question - the ticket references a `placeholder` prop. Is that meant for the visible, masked, text that appears above the input? If so, I was thinking of using something like `helpText` instead since `placeholder` has another meaning.

Reply:
I don't think we'll need a placeholder prop if we craft our masking function to print the unfilled mask when it's given an empty string. For example, if we're masking a date:
`myMaskFn("")` =`DD/MM/YYYY`
`myMaskFn("1.02")` = `01/02/YYYY`

Reply:
`myMaskFn("1/2/2022")` = `01/02/2022`
`myMaskFn("01-02-2022")` = `01/02/2022`
`myMaskFn("0201")` = `01/02/YYYY`